src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -226,7 +266,7 @@ export default {
 		},
 		// TODO: Review all tracking cateogries
 		onInteractCarousel(interaction) {
-			this.$kvTrackEvent('carousel', 'click-carousel-horizontal-scroll', interaction);
+			this.$kvTrackEvent('sidebar-carousel', 'swipe-carousel-horizontal-scroll', interaction);
```

Comment:
Let's leave this event as it was, and we can discuss tracking details for this experiment in a different ticket.

Reply:
Thank you